app/javascript/packs/SchoolsCoursesCurriculumPack.res

```diff
@@ -7,6 +7,7 @@ type props = {
   targetGroups: list<TargetGroup.t>,
   targets: list<Target.t>,
   hasVimeoAccessToken: bool,
+  vimeoAccountPlan: string,
```

Comment:
Wouldn't it be better to handle the possible string values here at the _edge_, and convert the incoming value to a variant (`VimeoPlan.t`), instead of _switching_ on a string in the `ContentBlockCreator` component?

There's a non-zero likelihood of errors creeping in when switching on the same possible string values more than once. 😅